packages/service-configuration/service-configuration.d.ts

```diff
@@ -5,6 +5,12 @@ export interface Configuration {
   secret: string;
 }
 
+export interface ConfigError extends Error {
+  constructor(serviceName?: string);
+  message: string;
+}
+
```

Comment:
Hmm. This should probably be declared as a class, not as an interface

Reply:
Thanks, will apply that change. But a side question, whenever I make changes in TS definition file vscode isn't picking up on them, any idea why?